tests/figures/chart/charts_component.test.ts

```diff
@@ -1056,7 +1056,12 @@ describe("charts", () => {
 
     test("Side panel correctly reacts to has_header checkbox check/uncheck (with only one point)", async () => {
       createTestChart("basicChart");
-      updateChart(model, chartId, { type: "line", labelRange: "C2", dataSets: ["A1"] });
+      updateChart(model, chartId, {
+        type: "line",
+        labelRange: "C2",
+        dataSets: ["A1"],
+        dataSetsHaveTitle: false,
```

Comment:
Why do we need this now ? Was it a mistake not to have this property set here before ?

Reply:
Yeah we had a dataset of one cell with dataSetsHaveTitle = true ... there's no dataset there ...